packages/site/src/components/ComponentLinks.tsx

```diff
@@ -9,16 +10,68 @@ import { useAtlantisSite } from "../providers/AtlantisSiteProvider";
  */
 export const ComponentLinks = ({
   links,
+  goToProps,
+  goToUsage,
+  goToDesign,
 }: {
   readonly links?: ContentExportLinks[];
+  readonly goToProps: () => void;
+  readonly goToUsage: () => void;
+  readonly goToDesign: () => void;
 }) => {
+  const [hlinks, setHlinks] = useState<Element[] | null>(null);
+
+  useEffect(() => {
+    const hdd = document.querySelectorAll("[data-heading-link]");
+
+    if (hdd.length > 0) {
+      setHlinks(Array.from(hdd));
+    }
+  }, []);
+
+  const click = (e: MouseEvent) => {
+    e.preventDefault();
+    const id = e.currentTarget?.getAttribute("href")?.replace("#", "");
+
+    if (id) {
+      goToDesign();
+      setTimeout(() => {
+        const element = document.getElementById(id);
+
+        if (element) {
+          element.scrollIntoView({ behavior: "smooth" });
+        }
+      }, 100);
+    }
+  };
   const { isMinimal } = useAtlantisSite();
   if (isMinimal) return null;
 
   return (
     <Content>
       <Box padding="base" direction="column">
-        <Heading level={2}>Links</Heading>
+        <Box margin={{ bottom: "base" }}>
+          <Heading level={3}>Design</Heading>
+          {hlinks?.map((link, index) => (
+            <Box key={index}>
+              <a onClick={click} href={`#${link.id}`}>
+                {link.textContent}
+              </a>
+            </Box>
+          ))}
+        </Box>
+        <Box margin={{ bottom: "base" }}>
+          <Heading level={3}>Implementation</Heading>
+          <Box>
+            <a onClick={goToUsage} href={`#`}>
+              Usage
+            </a>
+            <a onClick={goToProps} href={`#`}>
+              Props
+            </a>
+          </Box>
+        </Box>
+        <Heading level={3}>Links</Heading>
         <Box>
           {links?.map((link, index) => (
             <Box key={index}>
```

Comment:
These external links should also have the Link icon:
<img width="638" alt="image" src="https://github.com/user-attachments/assets/2b9103d4-c45d-4e07-a6d0-8a679925bc05">

Reply:
Done!